### DISCUSSION TITLE
Unable to install medspacy windows

### DISCUSSION THREAD

**Discussion by cha-vivianmakrch** [medspacy/medspacy#251] [General]
Hi there,
Any help is appreciated pls 
Unable to install due to error: 
ERROR: Failed building wheel for unqlite

 error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1

 error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1

Current python version: Python 3.11.1 (tags/v3.11.1:a7a450f, Dec  6 2022, 19:58:39) [MSC v.1934 64 bit (AMD64)] on win32
Screenshot:
![image](https://github.com/medspacy/medspacy/assets/115725773/417caa9d-10d8-4a31-9d34-bade9115f6c2)

![image](https://github.com/medspacy/medspacy/assets/115725773/504c4e5a-d4ce-4fea-8ecc-15b3f921df76)

**Comment by burgersmoke** [comment]
It looks like from this output that you are getting this when running this command:

`setup.py bdist_wheel`

And I think this is why the root error here is "Microsoft Visual C++ 14.0 or greater is required...."

If you do not have a compatible C++ compiler installed, this will be the error you get because some of the dependencies here involve compiling C++ code.

Do you get errors when you try to install medspacy with pip instead?

`pip install medspacy`

**Reply by cha-vivianmakrch** [reply]
HI there
yes this is the error i got when i do pip install medspacy
how do I get a compatible C++ compiler installed? 
![image](https://github.com/medspacy/medspacy/assets/115725773/24f44256-c171-4f80-9ac1-68bbc82c866c)

**Comment by burgersmoke** [comment]
The output provides a link of where to get the most common compiler for Windows:

https://visualstudio.microsoft.com/visual-cpp-build-tools/
